src/renderer/i18n/de/halt.ts

```diff
@@ -3,8 +3,8 @@ import { HaltMessages } from '../types'
 const halt: HaltMessages = {
   'halt.thorchain': 'THORChain wurde vorübergehend angehalten.',
   'halt.trading': 'Der Handel ist vorübergehend für alle Pools gestoppt.',
-  'halt.trading.eth': 'Der Handel für ETH ist vorübergehend gestoppt.',
-  'halt.chain': 'Der Handel in Pools von {chain} Chain(s) ist gestoppt aufgrund von Wartungsarbeiten.',
+  'halt.chain': 'Der Handel in Pools von {chains} Chain(s) ist gestoppt aufgrund von Wartungsarbeiten.',
```

Comment:
`Der Handel in Pools von {chains} Chain(s) wurde aufgrund von Wartungsarbeiten vorübergehend gestoppt.`

Reply:
Will be added by #1727